alcs-api/src/user/user.dto.ts

```diff
@@ -1,7 +1,16 @@
 import { AutoMap } from '@automapper/classes';
-import { IsEmail, IsString } from 'class-validator';
+import { Type } from 'class-transformer';
+import { IsDefined, IsEmail, IsOptional, IsString } from 'class-validator';
+
+export class UserSettingsDto {
+  favoriteBoards: string[];
+}
 
 export class CreateOrUpdateUserDto {
+  @AutoMap()
```

Comment:
Since a user can't change uuid, and the lookup is by email, then this doesn't need to be here.